internal/wasmrun/cache/cache_test.go

```
@@ -0,0 +1,457 @@
+	if err := os.WriteFile(legacyPath, data, 0o600); err != nil {
+		t.Fatalf("write legacy wasm: %v", err)
+	}
+	if err := os.WriteFile(legacySumPath, []byte(fmt.Sprintf("%x", sum[:])), 0o600); err != nil {
+		t.Fatalf("write legacy sum: %v", err)
+	}
+
+	rc, err := c.Get(h)
+	if err != nil {
+		t.Fatalf("Get(legacy) error = %v", err)
+	}
+	defer rc.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `rc.Close` is not checked

(errcheck)